src/NetPanzer/Core/main.cpp

```diff
@@ -242,12 +242,11 @@ BaseGameManager* initialise(int argc, char** argv) {
     exit(1);
   }
 
-#ifdef NP_DATADIR
+  Package::assignDataDir();
   try {
-    filesystem::addToSearchPath(NP_DATADIR);
+    filesystem::addToSearchPath(Package::getDataDir().c_str());
```

Comment:
See, I thought I could just change this and forget about using getSubPath() but it made no difference. netpanzer will still only run from the source root. I'm not sure how that "addToSearchPath" works anyway. To me, it sounds like the program would be searching different paths throughout the life of the program. e.g look here for a map, if it's not there, look for it. Then next time it has to look for maps, it would look through each "searchpath". But I dunno.

Reply:
My goal is to have netpanzer look for the correct data dir once and not ever have to do it again.